packages/smooth_app/lib/pages/scan/continuous_scan_page.dart

```diff
@@ -21,93 +22,105 @@ class _ContinuousScanPageState extends State<ContinuousScanPage> {
+                    overlayColor: Colors.transparent,
+                    // This offset adjusts the scanning area on iOS.
+                    cutOutBottomOffset: viewFinderBottomOffset,
+                  ),
+                  key: _scannerViewKey,
+                  onQRViewCreated: model!.setupScanner,
                 ),
-                key: _scannerViewKey,
-                onQRViewCreated: model!.setupScanner,
               ),
-            ),
-            SmoothRevealAnimation(
-              delay: 400,
-              startOffset: const Offset(0.0, 0.1),
-              animationCurve: Curves.easeInOutBack,
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.start,
-                children: <Widget>[
-                  Padding(
-                    padding: qrScannerPadding,
-                    child: SmoothViewFinder(
-                      boxSize: scannerSize,
-                      lineLength: screenSize.width * 0.8,
+              SmoothRevealAnimation(
+                delay: 400,
+                startOffset: const Offset(0.0, 0.1),
+                animationCurve: Curves.easeInOutBack,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: <Widget>[
+                    Padding(
+                      padding: qrScannerPadding,
+                      child: SmoothViewFinder(
+                        boxSize: scannerSize,
+                        lineLength: screenSize.width * 0.8,
+                      ),
                     ),
-                  ),
-                ],
+                  ],
+                ),
               ),
-            ),
-            SmoothRevealAnimation(
-              delay: 400,
-              startOffset: const Offset(0.0, -0.1),
-              animationCurve: Curves.easeInOutBack,
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.start,
-                children: <Widget>[
-                  _buildButtonsRow(context, model!),
-                  const Spacer(),
-                  SmoothProductCarousel(
-                    showSearchCard: true,
-                    height: carouselHeight,
-                  ),
-                ],
+              SmoothRevealAnimation(
+                delay: 400,
+                startOffset: const Offset(0.0, -0.1),
+                animationCurve: Curves.easeInOutBack,
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: <Widget>[
+                    _buildButtonsRow(context, model!),
+                    const Spacer(),
+                    SmoothProductCarousel(
+                      showSearchCard: true,
+                      height: carouselHeight,
+                    ),
+                  ],
+                ),
               ),
-            ),
-          ],
-        ),
-      );
-    });
+            ],
+          ),
+        );
+      }),
```

Comment:
What about an extra comma, like `},),`, in order to find the previous format/tabulation?